### PR TITLE
Release 1.4.1

### PR DESCRIPTION
**Pull request by dweitzman-codaio**
Mostly so https://github.com/coda/packs-sdk/pull/2657 can be released to packs studio

### CODE REVIEW

**Comment by ekoleda-codaio** [comment]
Was this PR created by the `make release-manual` step?

**Comment by jonathan-codaio** [comment]
LGTM assuming this is going through our release tooling

**Comment by dweitzman-codaio** [comment]
> Was this PR created by the `make release-manual` step?

It wasn't, but I'm re-creating it so the ordering and github link fixes in `CHANGELOG.md` are manually created but everything else will be created by `make release-manual`

**Comment by ekoleda-codaio** [comment]
> but I'm re-creating it

As in your are creating a different PR? Or re-doing this one? 

> ordering and github link fixes in CHANGELOG.md are manually created

The release process should handle all the `CHANGELOG.md` updates automatically.

**Comment by dweitzman-codaio** [comment]
> > but I'm re-creating it
> 
> As in your are creating a different PR? Or re-doing this one?
> 
> > ordering and github link fixes in CHANGELOG.md are manually created
> 
> The release process should handle all the `CHANGELOG.md` updates automatically.

The bottom of `CHANGLOG.md` is currently inconsistent and has some links with base tags that seem unhelpful, so I tried to fix it up manually. I'm trying to figure out in a separate branch why the bottom of our changelog (created by https://github.com/release-it/keep-a-changelog) doesn't match the example changelog on https://keepachangelog.com/en/1.1.0/ in terms of sort order

File: CHANGELOG.md [hunk 1]
```diff
@@ -589,11 +591,16 @@ await myHelper(context);
 
 - Beginning of alpha versioning.
 
-[unreleased]: https://github.com/coda/packs-sdk/compare/v1.4.0...HEAD
-[1.2.3]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.3
-[1.2.2]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.2
-[1.2.1]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.1
-[1.2.0]: https://github.com/coda/packs-sdk/compare/v1.0.1...v1.2.0
+[unreleased]: https://github.com/coda/packs-sdk/compare/v1.4.1...HEAD
```
Comment (dweitzman-codaio) [inline CHANGELOG.md:594]: I sorted this to match the order of the example at https://keepachangelog.com/en/1.1.0/ and fixed many of the base comparison version

Comment (dweitzman-codaio) [reply CHANGELOG.md:594]: I see the base comparison version added by release-it for this release was also wrong, which might explain why so many others were wrong. `release-it` picked v1.0.1 as the comparison base for v1.4.1 when it should have been v1.4.0

Comment (dweitzman-codaio) [reply CHANGELOG.md:594]: I've fixed this manually. After the release I'll see if `release-it` can continue doing the correct thing in the future after this PR or if maybe we need some kind of a config or command line argument change

Comment (dweitzman-codaio) [reply CHANGELOG.md:594]: I verified in another branch that after this PR `release-it` will be able to correctly update the changelog in the future

**Review by jonathan-codaio — APPROVED** [review]

**Pull request merged**